### PR TITLE
ci: updated RP tests (in sync with used version)

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: ci/tests/rp/test.sh [hunk 1]
```diff
@@ -8,14 +19,12 @@ echo "--- smoke test $ret"
 
 echo '--- unit test'
 pytest -vvv tests/unit_tests
-test "$ret" = 0 && ret=$?
+test "$ret" = 0 || ret=$?
```
Comment (andre-merzky) [inline ci/tests/rp/test.sh:22]: I don't understand that change: if a previous test suceeded (`ret=0`) and this one failed (`$?=1`), then the new version will ignore that failure:
```sh
$ ret=0
$ test "$ret" = 0 || ret=1
$ echo $ret
0
```

**Review by andre-merzky — CHANGES_REQUESTED** [review]
The RP version change looks good to me, but please check the comment about the shell logic change.

File: ci/tests/rp/test.sh (hunk 1 above)
Comment (mtitov) [reply ci/tests/rp/test.sh:22]: right.. should we just remove `test "$ret" = 0 ||`? and only get return value to print it out

Comment (andre-merzky) [reply ci/tests/rp/test.sh:22]: I think this should be left as is.  The rationale is that we want to run all tests, but if any one of them returns a non-zero exit code, then that will be used as the final result.  The current code does just that. (`exit $ret`).

File: ci/tests/rp-parsl/test.sh [hunk 2]
```diff
@@ -14,4 +14,4 @@ python map_reduce.py
 ret=$?
 echo "--- smoke test $ret"
 
-exit $ret
+echo "Success!"
```
Comment (andre-merzky) [inline ci/tests/rp-parsl/test.sh:17]: Same here btw - we do want the exit code to propagate...

File: ci/tests/rp/test.sh (hunk 1 above)
Comment (mtitov) [reply ci/tests/rp/test.sh:22]: Ok, got it! Thanks!

File: ci/tests/rp-parsl/test.sh (hunk 2 above)
Comment (mtitov) [reply ci/tests/rp-parsl/test.sh:17]: Ok, thought to keep in sync with other tests - in a way to collect return code (status of tests running), but not to set a corresponding code for the script itself (thus, to distinguish failed script and failed tests)

**Review by andre-merzky — APPROVED** [review]
LGTM - thanks!

**Pull request merged**